app/assets/stylesheets/era_custom.css.scss

```diff
@@ -488,6 +488,19 @@ body.home h4{
 	color: $link1Hover;
 }
 
+.avalon>a{
```

Comment:
This file is SASS friendly, lets use SASS?

You can rewrite the following as this instead:

```
.avalon > a {
  color: $link1;
  &:visited {
    color: $link1Visited;
  }
  &:hover, &:visited:hover {
    color: $link1Hover;
  }
}
```

Also, why the child selector? Is this needed? And do you want every `.avalon` class anchor tag to have this behaviour? Or is this just for the footer?